bluemira/radiation_transport/advective_transport.py

```diff
@@ -231,12 +231,31 @@ def _make_flux_surfaces_ib(self):
         self.flux_surfaces_ib_lfs = []
         self.flux_surfaces_ib_hfs = []
         x = self.x_sep_imp - self.dx_mp
-        while x > x_out_imp:
+        while x > x_out_imp + EPS:
```

Comment:
All good for me. Just here, does this epsilon actually work? Sometime it's too small and it doesn't make any difference.

Reply:
It doesn't fix this problem, no,  or at least not for the inputs I received. But it could help in a situation where the first wall radius is approx equal to x, and give a small mathematical buffer to prevent flux surfaces being generated outside (or parallel and coincident with) the first wall.

If you're happy, you can press review -> approve changes; as you are a reviewer for this module it will then make it possible for the PR to be approved.